src/components/Rockets/Rockets.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from 'react';
+import { Box } from '@mui/material';
+import { useDispatch, useSelector } from 'react-redux';
+import Rocket from './Rocket/Rocket';
+import { fetchRockets } from '../../redux/rockets/rockets';
+import styles from './styles';
+
+const Rockets = () => {
+  const dispatch = useDispatch();
+  const selector = useSelector((state) => state.rockets);
+
+  useEffect(() => {
+    dispatch(fetchRockets());
+  }, []);
```

Comment:
- [ ] Kindly make sure data should not be added to the store on every re-render (i.e. when changing views / using navigation). I highly recommend to only fetch the data again only I the mission is empty because at this moment the data is fetched whenever we navigate to another view.

_**Suggestion**_
` useEffect(() => {
    if (selector.length === 0) {
      dispatch(fetchRockets());
    }
  }, []);
`